ceph/ceph_admin/apply.py

```diff
@@ -122,12 +122,16 @@ def apply(self: ServiceProtocol, config: Dict) -> None:
         if args:
             base_cmd.append(config_dict_to_string(args))
 
-        self.shell(args=base_cmd)
+        out, err = self.shell(args=base_cmd)
+
+        if not out:
```

Comment:
For apply, there will be an output... otherwise, we would not be able to get the name of the service that was scheduled. If it is empty, it should error out